forge-gui/res/cardsfolder/upcoming/the_creation_of_avacyn.txt

```diff
@@ -0,0 +1,10 @@
+Name:The Creation of Avacyn
+ManaCost:1 B B
+Types:Enchantment Saga
+K:Chapter:3:I,II,III
+SVar:I:DB$ ChangeZone | ChangeNum$ 1 | ChangeType$ Card | Mandatory$ True | Origin$ Library | Destination$ Exile | ExileFaceDown$ True | WithMayLook$ True | SpellDescription$ Search your library for a card, exile it face down, then shuffle.
```

Comment:
what makes you think the extra MayLook is allowed? :P

Reply:
Heh because it's my card and I exiled it?

But I don't know how to play MtG or care to learn apparently 😅